GVFS/GVFS.Platform.Mac/MacDaemonLister.cs

```diff
@@ -0,0 +1,74 @@
+using GVFS.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace GVFS.Platform.Mac
+{
+    /// <summary>
+    /// Class to query the configured services on macOS
+    /// </summary>
+    public class MacDaemonLister

```

Comment:
I'm still not liking this name.. "Lister". What is the aversion to "Manager" or "Controller"?
At the moment all we're implementing is 'list', but why not 'stop' or 'restart'? You wouldn't expect a "MacDaemonStopper" class.

Reply:
Yeah... I don't really have an aversion to any of these names...

I wanted this to be semi-similar to [ServiceController](https://docs.microsoft.com/en-us/dotnet/api/system.serviceprocess.servicecontroller?view=netframework-4.8) in spirit, which seems to point towards `MacDaemonController` or `DaemonController` if it were targeting at POSIX.

I don't think we have any immediate need to flesh it out the functionality at this point... I am fine with either its current name for current functionality, or with a more aspirational name for if we ever expand it.